internal/modules/configuration/uploads_describer.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright (c) 2019 the Octant contributors. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package configuration
+
+import (
+	"context"
+
+	"github.com/vmware-tanzu/octant/internal/describer"
+	"github.com/vmware-tanzu/octant/internal/octant"
+	"github.com/vmware-tanzu/octant/pkg/view/component"
+)
+
+// UploadsDescriber describes an upload
+type UploadsDescriber struct {
+}
+
+var _ describer.Describer = (*UploadsDescriber)(nil)
+
+// Describe describes the upload yaml interface
+func (d *UploadsDescriber) Describe(ctx context.Context, namespace string, options describer.Options) (component.ContentResponse, error) {
+	title := append([]component.TitleComponent{}, component.NewText("Upload YAML"))
+	editor := component.NewEditor(component.TitleFromString("YAML"), "", false)
+	editor.Config.SubmitLabel = "Apply"
+	editor.Config.SubmitAction = octant.ActionApplyYaml
+	list := component.NewList(title, []component.Component{editor})
+
+	return component.ContentResponse{
+		Components: []component.Component{list},
+	}, nil
+}
+
+func (d *UploadsDescriber) PathFilters() []describer.PathFilter {
+	filter := describer.NewPathFilter("/upload", d)
```

Comment:
I think I'd prefer all of the references to be Apply to match convention. I can see the "upload YAML to apply it" but the way the UI functions, pasting it in to a box, that feels more like, Apply where a file picker would feel like Upload.

Reply:
I started off with "Upload" because their maybe other mechanism besides "apply". But since that's all we have, I agree "apply" makes more sense.